.github/workflows/pr-test.yml

```
@@ -0,0 +1,46 @@
+name: Test with Gradle
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)